publish-solution/action.yml

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+name: 'publish-solution'
+description: 'Power Platform Publish Solution Customizations'
+inputs:
+  environment-url:
+    description: 'URL of Power Platform environment to connect with; e.g. "https://test-env.crm.dynamics.com"'
+    required: true
+
+  user-name:
+    description: 'Power Platform user name, e.g. myname@my-org.onmicrosoft.com'
+    required: true
+
+  password-secret:
+    description: 'Power Platform password. Do NOT checkin password, instead create a secret and reference it here with: see: https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets#using-encrypted-secrets-in-a-workflow'
+    required: true
+
+runs:
+  using: 'node12'
+  # main: '../dist/actions/publish-solution/index.js'
```

Comment:
since your PR branch now has the updated dist/ with your new action, switch the action manifest back to using dist/
Otherwise, this action will fail if referenced from a different repo (out/ only exists in CI builds within this repo)